src/Codeception/Test/Loader/Unit.php

```diff
@@ -43,13 +43,7 @@ public function getTests()
 
     protected function createTestFromPhpUnitMethod(\ReflectionClass $class, \ReflectionMethod $method)
     {
-        if (method_exists(\PHPUnit\Framework\TestSuite::class, 'isTestMethod')) {
-            //PHPUnit <8.2
-            if (!\PHPUnit\Framework\TestSuite::isTestMethod($method)) {
-                return;
-            }
-            $test = \PHPUnit\Framework\TestSuite::createTest($class, $method->name);
-        } elseif (method_exists(\PHPUnit\Util\Test::class, 'isTestMethod')) {
+        if (method_exists(\PHPUnit\Util\Test::class, 'isTestMethod')) {
```

Comment:
This condition is not necessary, `\PHPUnit\Util\Test::isTestMethod` exists in all supported versions of PHPUnit.

Reply:
Yes, I thought I would leave it like that in case it disappears in the future.
But if you don't see it necessary, I'll remove it, the less the better.